.github/workflows/smoke-test.yml

```
@@ -14,7 +14,15 @@ defaults:
 jobs:
 
   build:
-    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        runs:
+          - arch: amd64
+            os: ubuntu-latest
+          - arch: arm64
+            os: ubuntu-24.04-arm
+    runs-on: ${{ matrix.runs.os }}
```

```suggestion
      matrix:
        runner:
          - arch: amd64
            os: ubuntu-latest
          - arch: arm64
            os: ubuntu-24.04-arm
    runs-on: ${{ matrix.runner.os }}
```
Resulting in more meaningful input names:
- `runner.os`
- `runner.arch`

To be changed below as well.

---

Done here: https://github.com/canonical/docker-snap/pull/215/commits/fddb35d9e7a1051a9793f18064483a13b7afeb92 thank you!